meta.yml

```diff
@@ -13,6 +13,7 @@ description: |-
   ALEA is a library for reasoning on randomized algorithms
   in Coq, based on interpreting programs inside a monad
   as probability distributions.
+  Also you can visit old ALEA repo: https://github.com/coq-contribs/random/
```

Comment:
```suggestion
  This repository used to be maintained at [coq-contribs/random](https://github.com/coq-contribs/random).
```

Reply:
Let's also add the link to the ALEA's webpage at https://www.lri.fr/~paulin/ALEA.